### PR TITLE
SIG Software Supply Chain Proposal

### PR DESCRIPTION
**Pull request by fdegir**

### CODE REVIEW

**Comment by tdcox** [comment]
I would like to see some clarity on how the remit of this group differs from that of Interoperability?

**Comment by oleg-nenashev** [comment]
Maybe it also makes sense to rename the Security SIG instead. It is no longer active AFAICT

**Comment by fdegir** [comment]
Thanks for the comments @tdcox and @oleg-nenashev.

> I would like to see some clarity on how the remit of this group differs from that of Interoperability?

The Interoperability SIG studies the seamless working of CI/CD technologies with focus being on terminology, (standardized) interfaces, and metadata.

The SIG Software Supply Chain aims to look at CI/CD aspects of the supply chain by studying various activities that are run as part of the pipelines and orchestrated by CI/CD technologies to ensure security, integrity, and compliance of the products and the pipelines themselves.

Some of the topics this SIG could potentially work on are SBOM and policy and these topics were discussed in SIG Interoperability in the past. However, the focus of SIG Interoperability was around interactions between CI/CD technologies and technologies provide SBOM capabilities and policy frameworks. This is one of the reasons for proposing SIG Software Supply Chain because these topics need to be looked into from how they can be employed to help with improving supply chain as they are used by CI/CD pipelines. The SIG Software Supply Chain will collaborate with other CDF SIGs when it identifies areas that are relevant for them instead of tackling them itself.

> Maybe it also makes sense to rename the Security SIG instead, it has strong overlap in the Manifesto. It is no longer active AFAICT

The SIG aims to take a holistics approach and look at various topics to contribute to overall effort of improving the situation with supply chain from CI/CD perspective. Security is one of the critical areas to look into however we aim to look into integrity and compliance aspects as well thus the SIG is proposed as SIG Software Supply Chain.

About the SIG Security.
Security of the supply chain was one of the sub areas the SIG Security together with SBOM so SIG Software Supply Chain will study the existing work of the SIG Security on these topics and look for possibilities to incorporate them. If community members desire to have workstreams focused on specific topics such as security of the supply chain, these efforts can be enabled by forming workstreams with the possibility of them turned into its own SIGs like how SIG Events was formed subject to CDF TOC review.

As an additional note, one of the purposes with this SIG is to bring the topic into attention of CDF Community by calling it out explicitly with a broader take since the topic is pretty critical however there is no active group working on this topic under CDF at the moment.

**Comment by tdcox** [comment]
Thanks for the clarification. My hesitation is due to the fact that it doesn’t feel aligned to the process of Continuous Delivery to treat ‘interoperability with other parts of the software supply chain’ and ‘interoperability with other parts of the software supply chain in a secure and congruent manner’ as two separate concerns.

**Comment by mjmckay** [comment]
I volunteer to be the TOC Sponsor for this SIG.

**Comment by fdegir** [comment]
> I volunteer to be the TOC Sponsor for this SIG.

Thanks for sponsoring the SIG @mjmckay. The proposal is update.

**Comment by mgreau** [comment]
Hi @fdegir 

I am interested in contributing to this SIG Software Supply Chain. Let me know how it works.

Thanks

**Comment by fdegir** [comment]
Thanks for your interest @mgreau. The SIG will need to be voted by the CDF TOC first which should happen soon and then we need to sort out the logistics (e.g. maillist, slack) if/when the SIG is approved by the TOC.

We will share updates on this PR so please keep an eye on it. In the meantime, I can add you as a member so please let me know.

**Comment by mgreau** [comment]
Thanks for the clarification.

Yes, please add me.

**Comment by majinghe** [comment]
Hi @fdegir 

I am also interested in the SIG, want to be one of contributor, will promote ideas and practice about Software Supply Chain Security in China.  Looking forwards the good news from the toc sides.

File: sigs/sig-software-supply-chain.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+* Liora Milbaum ([@lmilbaum](https://github.com/lmilbaum)), Red Hat
+* Fatih Degirmenci ([@fdegir](https://github.com/fdegir)), Ericsson Software Technology
+* Kara de la Marck ([@MarckK](https://github.com/MarckK)), CDF
+* Georg Kunz ([@gkunz](https://github.com/gkunz)), Ericsson
+* Erhan Vikyol ([@vikyol](https://github.com/vikyol)), Storebrand
+* Tracy Miranda ([@tracymiranda](https://github.com/tracymiranda)), Chainguard
+* Melissa McKay ([@mjmckay](https://github.com/mjmckay)), JFrog
+* Maxime Gréau ([@mgreau](https://github.com/mgreau)), Elastic
+* Majinghe ([@majinghe](https://github.com/majinghe))
```
Comment (todaywasawesome) [inline sigs/sig-software-supply-chain.md:39]:
```suggestion
* Majinghe ([@majinghe](https://github.com/majinghe))
* Dan Garfield ([@todaywasawesome](https://github.com/todaywasawesome)), Codefresh 
```

Comment (todaywasawesome) [reply sigs/sig-software-supply-chain.md:39]: @fdegir I love the idea and would like to volunteer Codefresh to be involved!

**Comment by todaywasawesome** [comment]
+1 non-binding

**Review by todaywasawesome — APPROVED** [review]

**Comment by fdegir** [comment]
The proposal is up for vote by CDF TOC on cdf-toc maillist. Thanks @oleg-nenashev for starting the vote.

https://lists.cd.foundation/g/cdf-toc/topic/88911382

/cc @cdfoundation/toc

File: sigs/@ [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (afrittoli) [inline sigs/@:1]: NIT: @fdegir I think this file may have been added by mistake?

Comment (fdegir) [reply sigs/@]: Thanks for the catch Andrea. Deleted the file now.

**Comment by lmilbaum** [comment]
@fdegir QQ Would it make sense to squash the commits?

**Comment by fdegir** [comment]
> @fdegir QQ Would it make sense to squash the commits?

Yes!
I was waiting to see if there are any further comments but given that we didn't receive any additional comments in TOC meeting and here, I think we can think of squashing and submitting later today.

**Review by oleg-nenashev — APPROVED** [review]
The request was voted on in the mailing list and then approved at the Feb 15 governance meeting. I will proceed with the merge https://lists.cd.foundation/g/cdf-toc/topic/88911382#775

**Pull request merged**